node/service/src/lib.rs

```diff
@@ -38,6 +37,10 @@ pub use sc_service::ChainSpec;
 use sc_transaction_pool_api::OffchainTransactionPoolFactory;
 pub use sp_api::ConstructRuntimeApi;
 
+/// The minimum period of blocks on which justifications will be
+/// imported and generated.
+const GRANDPA_JUSTIFICATION_PERIOD: u32 = 512;
```

Comment:
This change comes from substrate [PR #14423](https://github.com/paritytech/substrate/pull/14423)